Automattic/wp-calypso
# Block Editor Tracking: Remove "wpcom_block_editor_close_click" from W button

#### Changes proposed in this Pull Request

This is a companion PR to https://github.com/Automattic/wp-calypso/pull/53500

`wpcom_block_editor_close_click` is used to track when the editor is closed. A few months ago a navigation sidebar was implemented for the post editor, that time the W button's behaviour changed. It opened the navigation sidebar rather than closing the editor. To make sure we are tracking the right behaviour, this PR changes the following:

* Remove `wpcom_block_editor_close_click` from the WP button when it's supposed to open a navigation sidebar.

#### Testing instructions

* Follow steps to enable tracking debugging at PCYsg-nrf-p2
* Open post editor
* Click on the W button to open the navigation sidebar.
* Confirm "wpcom_block_editor_close_click" event is NOT triggered

E2E test will fail for Site Editor. 



Related to #53500 #53410
